File: src/main/java/com/pCarpet/model/Slot.java
package com.pCarpet.model;


import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.validation.constraints.NotNull;

import lombok.*;

@Data

@Entity
public class Slot {
	
	@Id
	@Column(name="id_slot")
	@GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
	
	@Column
	@NotNull
    private int number_carplace;
	
	@Column
	@NotNull
    private int number_carplace_free;
	
	@Column
    private double latitude;
	
	@Column
    private double longitude;
	
	@Column
    private String address;
	
	@Column
    private float price;
	
	@Column
    private String type;
	
	@ManyToOne
	@JoinColumn(name="username")
    private User user; 

}

File: PCarpet/Models/slot.cs
using PCarpet.DTO;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace PCarpet
{
    public partial class slot
    {
        public slot(SlotDTO slotDTO)
        {
            this.id = slotDTO.id;
            this.number_carplace = slotDTO.number_carplace;
            this.number_carplace_free = slotDTO.number_carplace_free;
            this.latitude = slotDTO.latitude;
            this.longitude = slotDTO.longitude;
            this.address = slotDTO.address;
            this.price = slotDTO.price;
            this.type = slotDTO.type;
            this.username = slotDTO.username;
        }

        public static SlotDTO toSlotDTO(slot slot)
        {
            return new SlotDTO(slot.id, slot.number_carplace, slot.number_carplace_free, slot.latitude, slot.longitude, slot.address, slot.price, slot.type, slot.username);
        }
    }
}